csrc/custom_all_reduce.cuh

```diff
@@ -23,17 +24,24 @@
 
 namespace vllm {
 
-constexpr int kMaxBlocks = 64;
-// note: we don't want to use atomics for signals because peer atomics are no
-// supported on PCIe links
+constexpr int kMaxBlocks = 36;
+// Using uint64_t to allow incrementing for long running tasks.
```

Comment:
does it matter? even if it overflows, I think it can still work. we just want to know if all ranks reach the same status.

Reply:
You're right. Unsigned int overflow is well-defined behavior.